### PR TITLE
Cards to Vue - all the rest

### PR DESCRIPTION
**Pull request by chosta**
It's done!
Thank you all for the help with reviews, advice and thanks for the patience. Took a while but I'm sure it's worth it :)
As next steps, I'll add a couple of PRs to refactor some code that solves issues like horizontal and vertical space. (you'll notice the bbbrrrrs, too cold I guess). I'll add a PR with unit tests too. For now let's finish this and have it all done!
@bafolts @kberg @Lynesth @nwai90 @alrusdi @sienmich @ssimeonoff

### CODE REVIEW

**Comment by chosta** [comment]
@kberg before I forget. I think you should check the loading of the `BEGINNER_CORPORATION` card. The `getCardExpansionByName` does not fetch it and I didn't want to do a side quest and fix it at this moment.

**Comment by kberg** [comment]
> @kberg before I forget. I think you should check the loading of the `BEGINNER_CORPORATION` card. The `getCardExpansionByName` does not fetch it and I didn't want to do a side quest and fix it at this moment.

What do you mean by "check"?

**Comment by chosta** [comment]
> What do you mean by "check"?

You were dealing with the cards manifest and I thought, you might know more about the issue. But this is one of those times when it's faster for me to fix it instead of explaining. I'll handle it.

**Review by alrusdi — APPROVED** [review]

File: src/cards/community/Playwrights.ts [hunk 1]
```diff
@@ -86,4 +89,23 @@ export class Playwrights implements CorporationCard {
...
+      cardNumber: 'R40',
+      description: 'You start with 38 MC and 1 Energy production.',
+      renderData: CardRenderer.builder((b) => {
+        b.br.br;
+        b.megacredits(38).productionBox((pb) => pb.energy(1));
+        b.corpBox('action', (cb) => {
+          cb.effectBox((eb) => {
+            // TODO(chosta): find a reasonable way to represent "?" (alphanumeric maybe)
+            // use 1000 as an id to tell Vue to render the '?'
+            eb.megacredits(1000).startAction;
```
Comment (kberg) [inline src/cards/community/Playwrights.ts:103]: `megacredits(number|'?')`

File: src/cards/prelude/ValleyTrust.ts [hunk 2]
```diff
@@ -43,4 +44,19 @@ export class ValleyTrust implements CorporationCard {
...
+    public metadata: CardMetadata = {
+      cardNumber: 'R34',
+      description: 'You start with 37 MC. As your first action, draw 3 Prelude cards, and play one of them. Discard the other two.',
+      renderData: CardRenderer.builder((b) => {
+        b.br.br;
+        b.megacredits(37).nbsp.prelude().asterix();
+        b.corpBox('effect', (ce) => {
+          ce.effectBox((eb) => {
+            eb.science(1).played.startEffect.megacredits(-2);
+            eb.description('Effect: When you play an Science tag, you pay 2MC less for it.');
```
Comment (kberg) [inline src/cards/prelude/ValleyTrust.ts:57]: "a Science tag"

File: src/cards/promo/Splice.ts [hunk 3]
```diff
@@ -64,4 +67,26 @@ export class Splice implements CorporationCard {
...
+          ce.effectBox((eb) => {
+            eb.microbes(1).played.any.startEffect;
+            eb.megacredits(2).any.or().microbes(1).any.asterix();
+            eb.description(undefined);
+          });
+          ce.vSpace();
+          ce.effectBox((eb) => {
+            eb.microbes(1).played.any.startEffect;
+            eb.megacredits(2);
+            eb.description('Effect: when a microbe tag is played, incl. this, THAT PLAYER gains 2 MC, or adds a microbe to THAT card, and you gain 2 MC.');
```
Comment (kberg) [inline src/cards/promo/Splice.ts:87]: Is this description correct? It doesn't make sense to me. But oh well.

File: src/components/card/Card.ts [hunk 4]
```diff
@@ -3,7 +3,6 @@ import Vue from 'vue';
 import {IProjectCard} from '../../cards/IProjectCard';
 import {ICard} from '../../cards/ICard';
 import {BeginnerCorporation} from '../../cards/corporation/BeginnerCorporation';
-import {HTML_DATA} from '../../HTML_data';
```
Comment (kberg) [inline src/components/card/Card.ts:6]: YES.

File: src/components/card/CardCorporationLogo.ts [hunk 5]
```diff
@@ -75,6 +75,72 @@ export const CardCorporationLogo = Vue.component('CardCorporationLogo', {
         result += '<span class="card-factorum-logo">FACTORUM</span>';
       } else if (title === CardName.MANUTECH) {
         result += '<span class="card-manutech-logo"><span style="color:white;background:#e63900;text-shadow:none;padding-left:2px;">MA</span>NUTECH</span>';
```
Comment (kberg) [inline src/components/card/CardCorporationLogo.ts:77]: This can be so much cleaner with a template, and a v-if.

File: src/components/card/CardRenderItemComponent.ts [hunk 6]
```diff
@@ -31,7 +31,7 @@ export const CardRenderItemComponent = Vue.component('CardRenderItemComponent',
       } else if (type === CardRenderItemType.OCEANS) {
         classes.push('card-global-requirement');
         classes.push('card-ocean-global-requirement');
-        if (this.item.size !== undefined) {
+        if (this.item.size !== undefined && this.item.size !== CardRenderItemSize.MEDIUM) {
```
Comment (kberg) [inline src/components/card/CardRenderItemComponent.ts:34]: Why this change here and below?

**Review by kberg — APPROVED** [review]
Modulo some changes and questions that shouldn't stop you from merging this and resolving them later, even. Good work!

File: src/cards/promo/Splice.ts (hunk 3 above)
Comment (chosta) [reply src/cards/promo/Splice.ts:87]: It is :)

File: src/components/card/CardCorporationLogo.ts (hunk 5 above)
Comment (chosta) [reply src/components/card/CardCorporationLogo.ts:77]: On the list of improvements

**Comment by kberg** [comment]
🥳

**Comment by alrusdi** [comment]
Oh! @chosta you are a hero!
